.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "root": true,
+    "extends": "eslint:recommended",
+    "env": {
+        "node": true,
+        "es6": true
+    },
+    "parser": "babel-eslint",
+    "plugins": [
+        "flowtype"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module"
+    },
+    "rules": {
```

Comment:
nits:
* `no-extra-boolean-cast` doesn't seem to be required
* Would be clearer to use string value `"off"` for `no-console`
* `no-empty` would be closer to eslint:recommended with value `["error", { "allowEmptyCatch": true }]`, otherwise would use `"off"`